srcpkgs/pulseaudio/files/README.voidlinux

```diff
@@ -5,3 +5,4 @@ and security reasons.
 Optional dependencies:
 
 * `python3-PyQt5-dbus` for running `qpaeq`
+* `gst-plugins-bad1` and `libopenaptx` for aptX support
```

Comment:
Those are not actually optional dependencies since the package won't be build with them by default right?